src/components/ErrorMessage.tsx

```diff
@@ -0,0 +1,25 @@
+/** @jsx jsx */
+import { jsx, css } from "@emotion/react";
+import colors from "../colors";
+
+interface ErrorMessageProps {
+    show: boolean,
+    message: string
+}
+
+const styles = css`
+  color: ${colors.error};
+  font-size: 1.15rem;
+  line-height: 1.1rem;
+  margin: 1rem 0 0;
```

Comment:
This is still unresolved. This suggestion only works if you set the position, otherwise elements are still pushed down. Additionally, it might be a good idea to set a negative z-index to hide the element if it gets overlapped by something else like a menu.